zerver/lib/alert_words.py

```diff
@@ -36,26 +37,31 @@ def get_alert_word_automaton(realm: Realm) -> ahocorasick.Automaton:
     return alert_word_automaton
 
 def user_alert_words(user_profile: UserProfile) -> List[str]:
-    return ujson.loads(user_profile.alert_words)
+    alert_word_list = []
+    for alert in AlertWords.objects.filter(user_profile=user_profile):
+        alert_word_list.append(alert.word)
+    return alert_word_list
 
 def add_user_alert_words(user_profile: UserProfile, alert_words: Iterable[str]) -> List[str]:
     words = user_alert_words(user_profile)
 
     new_words = [w for w in alert_words if w not in words]
-    words.extend(new_words)
 
-    set_user_alert_words(user_profile, words)
+    for word in new_words:
+        AlertWords.objects.create(user_profile= user_profile, word= word)
```

Comment:
Please use a `.bulk_create` object here.  Also, the whitespace here is weird; please use PEP-8 standard Python style.